### DISCUSSION TITLE
Is the .m4b output file different to the .m4b file generated by AAX Audio Converter?

### DISCUSSION THREAD

**Discussion by RiosDeterioratingMentalHealth** [audiamus/BookLibConnect#137] [Q&A]
Book Lib Connect outputs a .m4b file of downloaded audiobooks by default. If I just want a .m4b file, is there any benefit to using AAX Audio Converter alongside Book Lib Connect? Does it do anything differently?

**Comment by audiamus** [comment]
The Book Lib Connect .m4b is the original Audible download with the AAC audio stream decrypted, but otherwise remains unchanged. All AAX Audio Converter .m4b output, split or not, will have the meta data processed, e.g. chapters adjusted or Audible proprietary tags replaced with common ones.

**Reply by philgoetz** [reply]
So, it's better to use AAX to download, and then Book Lib Connect to convert?

**Reply by audiamus** [reply]
AAX Audio Converter does not yet offer a download option of its own, so you should use BookLibConnect and activate the export option ther. Then feed the exported .aax file to AaxAudioConverter and you get the best of both.

**Reply by ckaotik** [reply]
Is there a way to just export using BookLibConnect? For this setup I end up with download + export, taking more time and more disk space. Or am I doing it wrong?

**Reply by CMos902** [reply]
@ckaotik Hi! Did you ever find a solution to this? I think I'm doing the same as you and ending up with two .M4B files. I like how AAXAudioConverter organizes directories and removes short chapters, but I don't like how I now have twice as much storage space used.

Do you, or @audiamus, know if I can just delete the contents of the Download and Export directories once AAXAudioConverter has converted the Export directory? I'm worried I'll end up having everything downloaded again whenever I sync my Audible library.

**Reply by audiamus** [reply]
The books in the download and export folders are kept there for convenience.

- Download folder: You might want to process downloaded books with a different tool or play the .m4b directly.
- Export folder: You may want to run AaxAudioConverter with different settings for the same book. I often run in twice, with different settings, for my car radio and for my mobile phone.

When you start BookLibConnect it will recheck both folders to update the book status in the local database and the result will be shown in the status column. So, if a book is gone from the download folder it will be marked as not downloaded and show the globe icon.

Resync alone will never download books, it only downloads the book meta data, to synchronize with the local database

**Comment by Silther** [comment]
but the audio quality should be the same?

**Reply by audiamus** [reply]
No audio processing takes place, no recoding, it's a mere copy operation.

**Comment by Anthuzad** [comment]
It seems that the aax export is not working for me in the newest version of BookLibConnect, are there any gotchas that I might be missing? I've chosen an export folder and checked the export option.

**Reply by audiamus** [reply]
What does the log file say?

**Reply by Anthuzad** [reply]
Thank you for pointing me to the log file, I didn't look closely and made the assumption that the app tracks the target directory from selecting the download directory - it doesn't. I had a folder structure from a previous installation and the export folder defaulted to User/USERNAME/Music/Audible/Exports - my structure was similar so I didn't notice until I looked at the full path in the logs. All is well :)
